Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Performance Issue**: The loop increment expression 'i = i * 2' in the method Even will lead to an infinite loop if the initial value of 'i' is 0. This is a critical performance issue as it can cause the program to hang. <br> **Fix**: Change the loop increment expression to 'i++' or another appropriate increment that ensures the loop can terminate. If the intention is to iterate over even indices, you should use 'i += 2'. <br> **Code Suggestion**: 
 ```
 for (int i = 0; i < arr.length; i += 2)
 {
     System.out.println(arr[i]);
 }
 ```